CRM/Core/BAO/Mapping.php

```diff
@@ -115,7 +115,7 @@ public static function add($params) {
    *   Array of mapping names, keyed by id.
    */
   public static function getMappings($mappingType) {
-    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('limit' => 1, 'sort' => 'name')));
+    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('sort' => 'name')));
```

Comment:
@mickadoo It may make more sense to do a separate pull-request once this is merged for the style changes so that it's clear that this was a bug fix.